packages/postcss/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # @webpack-blocks/postcss - Changelog
 
+## 1.0.0-beta.5
```

Comment:
`Next release` might be better. The next release could just as well be `1.0.0-rc`.